include/blmc_robots/teststand.hpp

```diff
@@ -12,6 +12,8 @@
 #include <blmc_robots/common_header.hpp>
 #include <blmc_robots/blmc_joint_module.hpp>
 #include <AtiFTSensor.h>
+#include <imu-core/imu_3DM_GX5_25.hpp>
+#include <stdio.h> 
```

Comment:
Do not use the stdio. Use the real_time_tools/iostream.hpp which provides you with the rt_printf function.